aquacomputer_d5next.c

```diff
@@ -182,6 +184,18 @@ static u16 quadro_ctrl_fan_offsets[] = { 0x36, 0x8b, 0xe0, 0x135 };
 #define HIGHFLOWNEXT_5V_VOLTAGE		97
 #define HIGHFLOWNEXT_5V_VOLTAGE_USB	99
 
+/* Register offsets for Leakshield */
+#define LEAKSHIELD_PRESSURE_ADJUSTED 285
+#define LEAKSHIELD_TEMPERATURE_1 265
+#define LEAKSHIELD_TEMPERATURE_2 287
+#define LEAKSHIELD_PRESSURE_MIN 291
```

Comment:
Hwmon has support for `fan[1-*]_min`, `fan[1-*]_max` and `fan[1-*]_target`. As pressure is already exposed as a fan, use these (the driver already uses min and max) to fit into sysfs. This will all have to be noted in the driver docs at some point (but I need to rewrite some parts there first, so it's not urgent).